user_sync/connector/umapi_util.py

```diff
@@ -3,10 +3,11 @@
 
 
 def make_auth_dict(name, config, org_id, tech_acct, logger):
+    api_field = 'client_id' if 'client_id' in config else "api_key"
     auth_dict = {
         'org_id': org_id,
-        'tech_acct_id': tech_acct,
```

Comment:
This line should not be changed - must be tech_acct_id since this is what umapi client expects.